platform/linuxbsd/x11/detect_prime_x11.cpp

```diff
@@ -60,6 +60,9 @@
 
 typedef GLXContext (*GLXCREATECONTEXTATTRIBSARBPROC)(Display *, GLXFBConfig, GLXContext, Bool, const int *);
 
+// To prevent shadowing warnings
+#undef glGetString
```

Comment:
If this gives a shadowing warning, it shows that we might not need this code at all and that glad provides it for us already:
```cpp
PFNGLGETSTRINGPROC glGetString = (PFNGLGETSTRINGPROC)glXGetProcAddressARB((GLubyte *)"glGetString");
if (!glGetString) {
        print_verbose("Unable to get glGetString, GPU detection skipped.");
        quick_exit(1);
}
```

Reply:
Then I'll try using it, thank you

Reply:
Well nevermind, I tried to compile without it and it's segfaulting with whatever `glGetString` is defined as without this.

Reply:
Oh thank you for catching it